components/prize_pool/commons/prize_pool_base.lua

```diff
@@ -82,13 +82,27 @@ BasePrizePool.config = {
 	storeSmw = {
 		default = true,
 		read = function(args)
-			return Logic.readBoolOrNil(args.storesmw)
+			local disabledVariable = Logic.readBoolOrNil(Variables.varDefault('disable_LPDB_storage'))
+			if disabledVariable ~= nil then
+				disabledVariable = not disabledVariable
+			end
+			return Logic.nilOr(
+				Logic.readBoolOrNil(args.storesmw),
+				disabledVariable
+			)
```

Comment:
Any suggestions to make this more readable?

Reply:
Maybe
```suggestion
			return Logic.nilOr(
				Logic.readBoolOrNil(args.storesmw),
				function()
					local variable = Logic.readBoolOrNil(Variables.varDefault('disable_LPDB_storage'))
					if variable ~= nil then
						return not disabledVariable
					end
					return nil
				end
			)
```

Reply:
Either works for me, can't think of a better one

Reply:
both fine for me

Reply:
Then i'll just leave it as is.